### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by oorion**
add bootstrap, articles, and article show page.

### CODE REVIEW

File: app/routes/article.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function(params){
+    return this.store.find('article', params.article_id);
```
Comment (oorion) [inline app/routes/article.js:5]: The route doesn't work with articleId

File: app/router.js [hunk 2]
```diff
@@ -6,4 +6,8 @@ var Router = Ember.Router.extend({
 });
 
 export default Router.map(function() {
+  this.resource('articles', { path: '/' }, function() {});
+  this.resource('articles', function() {
```
Comment (bmrsny) [inline app/router.js:10]: This is nice. Makes the url more readable!

File: app/templates/article.hbs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div class="col-md-4 article-show">
+  <h3>{{model.title}}</h3>
+  <p>{{model.abstract}}</p>
+</div>
```
Comment (bmrsny) [inline app/templates/article.hbs:4]: We may need to think   about our data and visualization more. So far the articles sidebar is doing the same as the article show page in content quality.

**Comment by bmrsny** [comment]
Looks good!! :tada:

**Pull request merged**